external/fv3fit/tests/reservoir/test_reservoir_computing_model.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, n_outputs: int):
     def predict(self, input):
         # returns vector of size n_outputs, with each element
         # the mean of the input vector elements
-        return np.full(self.n_outputs, np.mean(input))
+        return np.full(self.n_outputs, np.mean(input)).reshape(1, -1)
```

Comment:
This change is to replicate the behavior of the sklearn regressors, which predict 2d arrays.